src/transform/identifier-expression.js

```diff
@@ -1,9 +1,14 @@
+var die = require("../die");
 var es = require("../es");
 
 function IdentifierExpression(transform, node) {
     var id = transform(node.data);
     var name = es.Literal(node.data.data);
     var ref = es.Identifier("$ref");
+    if (ref === "_") {
```

Comment:
`ref` isn't the right thing to check for `"_"`. Add `console.error(ref)` and you can see what that object looks like and how  it doesn't vary.

Add a `console.error(node)` and you should see where the `"_"` will show up.